### ISSUE TITLE
BIOS Update option

### ISSUE THREAD

**Issue by generalvimal** [gokulmaxi/vantage#2]
I know this could be complicated. But is it possible to add bios update feature?

**Comment by gokulmaxi** [comment]
Yeah that would be a great option but currently I have no idea how to implement it.Any ideas or pull requests are welcome 🤗
